free-4.md

```diff
@@ -0,0 +1,82 @@
+# Free project 3
+
+## The user and a language
+This section describes who the project would serve and why a language might be a
+good way to meet their needs.
+
+
+### What's the need?
+_What need is met by your idea? Who are you helping? What is that person's
+experience like now? What would their experience be like if you could help 
+them?_
+
+Many programmers, especially novus programmers struggle to debug their code. They read the error message, but the only part the understand is that it means there code wont work. Especially for novus programmers, this can be an intensely frusterating process, they may only be missing a semicolon, of have used a ')' instead of '}' but they have dumb and inept, like computer science is something only "those smart people" can do. I would like to make a programming language that could be used along with existing programming language but would make the debugging process easier for programmers, by providing more easily understandable error messages with suggestions for what might be the problem.
+
```

Comment:
I really like that this project aims to open up the accessibility of CS to non-CS majors. I think trying to make the field less exclusive is a really important goal.